common.mk

```diff
@@ -129,10 +139,11 @@ $(output_dir)/%: $(RISCV)/riscv64-unknown-elf/share/riscv-tests/isa/%
 	ln -sf $< $@
 
 $(output_dir)/%.run: $(output_dir)/% $(sim)
-	(set -o pipefail && $(sim) $(PERMISSIVE_ON) +max-cycles=$(timeout_cycles) $(SIM_FLAGS) $(PERMISSIVE_OFF) $< </dev/null | tee $<.log) && touch $@
+	echo $(PROJECT_SIM_FLAGS)
+	(set -o pipefail && $(sim) $(PERMISSIVE_ON) +max-cycles=$(timeout_cycles) $(SIM_FLAGS) $(PROJECT_SIM_FLAGS) $(PERMISSIVE_OFF) $< </dev/null | tee $<.log) && touch $@
 
 $(output_dir)/%.out: $(output_dir)/% $(sim)
-	(set -o pipefail && $(sim) $(PERMISSIVE_ON) +dramsim +max-cycles=$(timeout_cycles) $(VERBOSE_FLAGS) $(PERMISSIVE_OFF) $< </dev/null 2> >(spike-dasm > $@) | tee $<.log)
+	(set -o pipefail && $(sim) $(PERMISSIVE_ON) +dramsim +max-cycles=$(timeout_cycles) $(VERBOSE_FLAGS) $(SIM_FLAGS) $(PROJECT_SIM_FLAGS) $(PERMISSIVE_OFF) $< </dev/null 2> >(spike-dasm > $@) | tee $<.log)
```

Comment:
I don't know what `dramsim` does but I think we should keep these two separate. 

In many cases, I have to add extra flags to the simulation with `SIM_FLAGS` so having these both move there will make it so that adding new variables also requires adding the `+dramsim +max-cycles` flags as well as `+abe-flag=1`.

A compromise that we can do is have the `SIM_FLAGS` use `override` and just add the `+dramsim +max-cycles` always? Thoughts?

Reply:
Yeah override is the solution for this.
`override SIM_FLAGS += +dramsim +max-cycles`

Reply:
Yup Ill change it to that then